manager/system_handler.py

```diff
@@ -782,6 +788,7 @@ def handle_get(cls, **kwargs):
                 .where(SystemPlatform.when_deleted.is_null(True))
                 .dicts()
             )
+            query = apply_filters(query, args, [filter_types.SYSTEM_TAGS], {})
```

Comment:
Suggestion to use kwargs, to make sure to understand what is the last param with empty dict.